dsc/debugResource.md

```diff
@@ -13,13 +13,16 @@ ms.prod:  powershell
 
 > Applies To: Windows PowerShell 5.0
 
-In PowerShell 5.0, a new feature was introduced in Desired State Configuraiton (DSC) that allows you to debug a DSC resource as a configuration is being applied.
+In PowerShell 5.0, a new feature was introduced in Desired State Configuraiton (DSC) that allows you to debug a DSC resource as a configuration 
+is being applied.

```

Comment:
You should be doing [semantic line feeds](http://rhodesmill.org/brandon/2012/one-sentence-per-line/) so that `git diff` is easy to read, i.e: break a paragraph at sentence end, not in the middle.